superset/db_engine_specs/presto.py

```diff
@@ -783,6 +783,7 @@ def _extract_error_message(cls, ex: Exception) -> Optional[str]:
     def _partition_query(  # pylint: disable=too-many-arguments,too-many-locals
         cls,
         table_name: str,
+        schema: Optional[str],
```

Comment:
Could you replace `table_name`/`schema` with the [`Table`](https://github.com/apache/incubator-superset/blob/master/superset/sql_parse.py#L56) dataclass.